docs/rackhd/install_os.rst

```diff
@@ -23,7 +23,7 @@ CoreOS        Graph.InstallCoreOS  899.17.0
 OS Installation Workflow APIs
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-Start an OS installation workflow:
+Start an OS installation workflow for CentOS:
```

Comment:
why change this to be specific to CentOS when the details are more generic? Or are you intending to set up additional documentation for every OS with these details?